### DISCUSSION TITLE
Annual Note Templater

### DISCUSSION THREAD

**Discussion by flowingblaze** [BryanJenksCommunity/FAQ#185] [Q&A] [answered]
Hello there! I tried to use your annual note template in your template vault and I got this error:
![image](https://github.com/BryanJenksCommunity/FAQ/assets/107226733/3d91168e-4aa5-4e00-a466-8448e4960f58)

how do I use it?

**Comment by tallguyjenks** [comment]
i'd need to see the information that was provided in the console in order to help with this

**Reply by flowingblaze** [reply, marked as answer]
I just opened obsidian to get the console error but for whatever reason It works now? I don't know what I changed to fix the issue but it is fixed somehow!

**Reply by wegjr** [reply]
Thank you for the inspirational content.

I was looking for your templates for your Daily Note Review System but don't see them here.

Are they still available for download?

**Reply by tallguyjenks** [reply]
@wegjr they are still available from my newsletter sign up or the latest versions of everything are available from signing up to be a github sponsor
